src/components/registerForm/RegisterForm.js

```diff
@@ -0,0 +1,20 @@
+import styles from "@/styles/logInModal.module.scss";
+
+const RegisterForm = ({setToLogin,setToAnonymous}) => {
+  return (
+    <>
+      <h3 onClick={setToLogin} className={styles.logInModalLink}>Logga in</h3>
```

Comment:
I'm wondering why you choose to use an h3 element here.

Reply:
Yeah I agree it isn't the best choice, I will change it 👍